packages/yande/src/index.ts

```diff
@@ -40,11 +40,15 @@ namespace YandeImageSource {
     endpoint: string
   }
 
-  export const Config: Schema<Config> = Schema.object({
-    label: Schema.string().default('yande').description('图源标签，可用于在指令中手动指定图源。'),
-    weight: Schema.number().min(1).default(1).description('图源权重。在多个符合标签的图源中，将按照各自的权重随机选择。'),
-    endpoint: Schema.string().description('yande 的 URL。').default('https://yande.re'),
-  })
+  export const Config: Schema<Config> = Schema.intersect([
+    Schema.object({
+      label: Schema.string().default('yande').description('图源标签，可用于在指令中手动指定图源。'),
+      weight: Schema.number().min(1).default(1).description('图源权重。在多个符合标签的图源中，将按照各自的权重随机选择。'),
+    }).description('全局设置'),
+    Schema.object({
+      endpoint: Schema.string().description('yande 的 URL。').default('https://yande.re'),
```

Comment:
```suggestion
      endpoint: Schema.string().description('Yande.re 的 URL。').default('https://yande.re'),
```

The actual name of this site should be `Yande.re`, as `Yandere` itself is a word but `yande` is not.